StereoKitC/stereokit.h

```diff
@@ -1413,6 +1413,7 @@ SK_API void          model_recalculate_bounds_exact(model_t model);
 SK_API void          model_set_bounds              (model_t model, const sk_ref(bounds_t) bounds);
 SK_API bounds_t      model_get_bounds              (model_t model);
 SK_API bool32_t      model_ray_intersect             (model_t model, ray_t model_space_ray, cull_ cull_mode, ray_t* out_pt);
+SK_API bool32_t      model_node_ray_intersect(model_t model, ray_t model_space_ray, cull_ cull_mode, ray_t* out_pt, model_node_id* model_node_id);
```

Comment:
I think in this case, it would be a little cleaner to add the new `model_node_id` parameter to `model_ray_intersect` instead of adding a new function? This can have a `sk_default(nullptr)` (with a corresponding null check in the implementation) to prevent this from being a "breaking" change. To indicate these things properly in SK's naming convention here in the public API, that would make the parameter name more like a `out_opt_node_id`.